NEWS.md

```diff
@@ -1,5 +1,9 @@
 # roxygen2 (development version)
 
+## Minor improvements and bug fixes
```

Comment:
Can you please remove this heading? As noted in https://style.tidyverse.org/news.html#news-release, we only add the subheads just prior to release.